### PR TITLE
Dynamic bottom sheet room header

### PR DESCRIPTION
**Pull request by AndyScherzinger**

### CODE REVIEW

File: app/src/main/java/com/nextcloud/talk/controllers/bottomsheet/CallMenuController.java [hunk 1]
```diff
@@ -123,7 +123,18 @@ private void prepareMenu() {
         menuItems = new ArrayList<>();
 
         if (menuType.equals(MenuType.REGULAR)) {
-            menuItems.add(new MenuItem(getResources().getString(R.string.nc_configure_room), 0, null));
+            if (room.displayName != null && room.displayName.length()>0) {
```
Comment (mario) [inline app/src/main/java/com/nextcloud/talk/controllers/bottomsheet/CallMenuController.java]: TextUtils.isEmpty(room.getDisplayName()) ? :)

Same in all other places.

Comment (AndyScherzinger) [reply app/src/main/java/com/nextcloud/talk/controllers/bottomsheet/CallMenuController.java]: fixed https://github.com/nextcloud/talk-android/pull/204/commits/2f3c2c0f17b39110e3187d8e70e2924eab89189e

**Comment by jancborchardt** [comment]
Yay @AndyScherzinger :kissing_heart:

**Comment by AndyScherzinger** [comment]
> Yay @AndyScherzinger :kissing_heart:

you are very welcome @jancborchardt @mario

**Comment by AndyScherzinger** [comment]
Build fails due to lint score which matches the masters actual lint score, one error due to translations which will be gone with the next transifex run _after_ the PR has been merged.

**Pull request merged**